test/cmdlineTests/asm_json_import_verbatim/output

```diff
@@ -0,0 +1,58 @@
+Opcodes:
+PUSH0 CALLDATALOAD PUSH1 0x14 DUP1 DUP3 SSTORE PUSH1 0x20 CALLDATALOAD ISZERO PUSH1 0x1C JUMPI DUP1 DUP3 SSTORE PUSH0 DUP1 PUSH0 DUP1 PUSH0 DUP1 STATICCALL POP PUSH25 0x797A5B80825550500000000000000000000000000000000000 
+EVM assembly:
+    /* "<stdin>":28:29  b */
+  0x00
+    /* "<stdin>":15:30      {... */
+  calldataload
+    /* "<stdin>":44:46     */
+  0x14
+    /* "<stdin>":61:62    */
+  dup1
+    /* "<stdin>":58:59    */
+  dup3
+    /* "<stdin>":51:63  : 29,... */
+  sstore
+    /* "<stdin>":84:86   " */
+  0x20
+    /* "<stdin>":71:87  "PUSH",... */
+  calldataload
+    /* "<stdin>":68:188  ": "PUSH",... */
```

Comment:
These code snippets should not be printed. The input file (`<stdin>`) is listed as one of the sources, which makes no sense but makes the compiler use it as the source file. The thing is, in the EVM asm import mode we never have access to sources so while printing locations makes sense, the snippets should be disabled. I.e. of all the values supported by `--debug-info` (`ast-id` `location` `snippet`) only `location` should be allowed. And the default should be different.